templates/base.html

```diff
@@ -47,6 +48,7 @@
                 </div>
               </div>
         </footer>
+        {% endblock footer %}
```

Comment:
Why is the footer in blocks? Same footer isn't applied to every page?

Reply:
I have added a footer block to the base template so that it can be overwritten in the grocery planning section where there is a footer that has an absolute position on the bottom of the page. In this situation the footer from base just gets in the way of the design and it doesn't seem necessary to keep a footer with just my personal information in the grocery planning section.